Cargo.toml

```diff
@@ -12,6 +12,20 @@ keywords = ["bitcoin", "paper", "wallet", "offline"]
 readme = "README.md"
 
 [dependencies]
-bitcoin = { version="0.25.2", features = ["rand"] }
+bitcoin = { version="0.26", features = ["rand"] }
+miniscript = "5.1.0"
```

Comment:
generally it's recommended to only pin up to the "minor" version, like `5.1`. patch-level releases are not supposed to change the interface, so the code should still compile but we could potentially get bugfixing or general improvements over time

Reply:
done in 2bad953